src/controller/CHIPDeviceControllerFactory.cpp

```diff
@@ -285,6 +285,9 @@ CHIP_ERROR DeviceControllerFactory::InitSystemState(FactoryInitParams params)
         // Consequently, reach in set the fabric table pointer to point to the right version.
         //
         app::DnssdServer::Instance().SetFabricTable(stateParams.fabricTable);
+
+        // Disable the TCP Server based on the TCPListenParameters setting in TransportMgr Init.
```

Comment:
Again, put the TcpListenParameters on the stack so we can use the boolean from it here.